internal/server/worker_list.go

```diff
@@ -71,10 +71,11 @@ func (w WorkerList) SupportsFeature(f version.Feature) WorkerList {
 // Shuffle returns a randomly-shuffled copy of the caller's Workers (using
 // crypto/rand). If the caller's WorkerList has one element or less, this
 // function is a no-op.
-func (w WorkerList) Shuffle() (WorkerList, error) {
+func (w WorkerList) Shuffle(opt ...Option) (WorkerList, error) {
```

Comment:
Normally when a function has an option, we document what options are supported:
```suggestion
// Supported options:
//   - WithRandomReader
func (w WorkerList) Shuffle(opt ...Option) (WorkerList, error) {
```

Reply:
comments added.